docs/README.md

```diff
@@ -60,6 +60,8 @@ MultiQC Modules:
     Salmon: modules/salmon.md
     STAR: modules/star.md
     TopHat: modules/tophat.md
+  Assemblers:
+    HiFiasm: modules/hifiasm.md
```

Comment:
I'm going to move this into an existing category, as I'm worried it'll break the website - sorry!

Currently in the process of rewriting the website and will remove these categories, which MultiQC outgrew a long time ago. Hoping to add keywords instead.